Amasty_Storelocator/templates/pages/location_view.phtml

```diff
@@ -0,0 +1,308 @@
+                                href="tel:<?= $block->escapeHtmlAttr($location->getPhone()) ?>"
+                                class="amlocator-link"
+                            >
+                                <?= $block->escapeHtml($location->getPhone()) ?>
+                            </a>
+                        </div>
+                    <?php endif; ?>
+                    <?php if (!empty($location->getEmail())): ?>
+                        <div class="amlocator-block">
+                            <span class="amlocator-icon -email"></span>
+                            <a
+                                href="mailto:<?= $block->escapeUrl($location->getEmail()) ?>"
+                                class="amlocator-link"
+                                title="<?= $block->escapeHtmlAttr($location->getName()); ?>"
+                            >
+                                <?= $block->escapeUrl($location->getEmail()) ?>
+                            </a>
+                        </div>
+                    <?php endif; ?>
+                </div>
+                <div class="amlocator-block">
+                    <a
+                        class="amlocator-link"
+                        data-amlocator-js="locator-panorama"
+                    >
+                        <?= $block->escapeHtml(__('Street View')) ?>
+                    </a>
+                </div>
+            </div>
+
+            <div class="amlocator-block">
+                <?= $block->getChildBlock('amasty_store_locator_schedule')->setData('location', $location)->toHtml(); ?>
+            </div>
+
+            <?php if (count($galleryImages)): ?>
+                <div class="amlocator-block -separator">
+                    <hr class="hr"/>
+                </div>
+                <div class="amlocator-block amlocator-location-gallery">
+                    <span class="amlocator-title">
+                        <?= $block->escapeHtml(__('Photos')) ?>
+                    </span>
+                    <div
+                        class="amlocator-wrapper"
+                        data-amlocator-js="locator-gallery"
+                    >
+                        <?php foreach ($galleryImages as $image): ?>
+                            <a
+                                class="amlocator-link"
+                                href="<?= $block->escapeUrl($image['path']) ?>"
+                                data-amlocator-js="location-image"
+                                data-fancybox="gallery"
+                            >
+                                <img
+                                    class="amlocator-image lazyload"
+                                    title="<?= $block->escapeHtmlAttr($location->getName()) ?>"
+                                    data-src="<?= $block->escapeUrl($image['path']) ?>"
+                                    alt="<?= $block->escapeHtmlAttr($location->getName()) ?>"
+                                />
```

Comment:
missing `src` attribute, we should use placeholder for it to avoid HTML validation issues. 
Placeholder for `src` attribute:
`data:image/gif;base64,iVBORw0KGgoAAAANSUhEUgAAAAoAAAABCAQAAABN/Pf1AAAAC0lEQVR42mNkwAIAACoAAgu1Hc4AAAAASUVORK5CYII&#x3D;`